object_detection/scripts/size_estimator.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+
+
+# import Locally used packages
+from math import sin, radians
+
+
+
+class SizeEstimatorClass():
+    fov_horizontal = 110.0                          # Degrees
+    fov_vertical = 70.0                             # Degrees
+    focal_length = 2.12                             # mm
+    max_width = 1344                                # pxl
+    max_height = 376                                # pxl
+    angles_pr_pxl_hor = fov_horizontal/max_width
+    angles_pr_pxl_ver = fov_vertical/max_height
+    use_single_lense = False
+
+    def main(self, bbox):
+        """
+        Takes in data from a bounding box message and calculates the size of the object.
+        
+        Args:
+            bbox                        the boundingbox message which includes data needed to calculate the size of an object
+
+        Returns:
+            size_estimator_data\t         list of usefull data\n
+                length_x_mtr     \t       width of object\n
+                length_y_mtr       \t     height of object\n
+                redefined_angle_x    \t   the horizontal angle to centre of boundingbox in a new coordinate frame shown below\n
+                redefined_angle_y      \t the vertical angle to centre of boundingbox in a new coordinate frame shown below\n
+
+        xy \t xy = -+ \t ++ \n
+        xy \t xy = -- \t +-\n
+
+        """
```

Comment:
Ref my above comment, this style of docstring is much better. Don't think the tabs and newline characters are needed though :smile: